source/plugins/go/input/lib/kubernetescontainerinventory.go

```diff
@@ -79,9 +79,15 @@ func GetContainerInventoryRecords(podItem map[string]interface{}, batchTime stri
 		if imageIDValue, ok := containerStatusMap["imageID"].(string); ok && imageIDValue != "" {
 			if atLocation := strings.Index(imageIDValue, "@"); atLocation != -1 {
 				containerInventoryRecord["ImageId"] = imageIDValue[atLocation+1:]
+			} else {
+				containerInventoryRecord["ImageId"] = imageIDValue
```

Comment:
any case: (e.g., repo@sha256 without :imageid)

Reply:
no this is not supported.